app/src/app/pages/plando-page/plando-page.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="logo-wrapper logo-paste-over">
+    <a class="image-link" [routerLink]="'../'">
+        <img src="../../../assets/images/plando/plandologo.png" class="logo">
+    </a>
+</div>
+<form [formGroup]="formGroup">
+    <div class="main-page">
+        <mat-card class="settings-card">
+            <mat-tab-group disablePagination="true" animationDuration="0" scrollToCenter>
+                <mat-tab label="Spirits & Chapters">
+                    <app-plando-spirits-and-chapters [plandoFormGroup]="formGroup"></app-plando-spirits-and-chapters>
+                </mat-tab>
+                <mat-tab label="Party">
+                    <app-plando-party [partnerMoveCostsFormGroup]="formGroup.get('move_costs').get('partner')"></app-plando-party>
+                </mat-tab>
+                <mat-tab label="Badges">
+                    <app-plando-badges [badgeMoveCostsFormGroup]="formGroup.get('move_costs').get('badge')"></app-plando-badges>
+                </mat-tab>
+                <mat-tab label="Items">
+                    <app-plando-items [itemsFormGroup]="formGroup.get('items')"></app-plando-items>
+                </mat-tab>
+            </mat-tab-group>
+        </mat-card>
+        <mat-card class="settings-card">
+            <div class="button-container">
+                <button [disabled]="formGroup.invalid || isValidating" class="submit" type="button" (click)="onSubmit()">
```

Comment:
A tooltip explaining there are errors when the button is disabled, like in the main rando page, would be nice